finetuner/client/exception.py

```diff
@@ -0,0 +1,24 @@
+from typing import Optional
+
+
+class BaseError(Exception):
+    response = None
+    code = -1
+    message = "An unknown error occurred"
+
+    def __init__(
+        self,
+        response: dict,
+        message: Optional[str] = None,
```

Comment:
setting default None values makes the code above more complex. Just set the class attribute values as default here:
```
class BaseError(Exception):

def __init__(
        self,
        response: dict,
        message: str = "An unknown error occurred",
        code: int = -1,
```